zerver/tests/test_home.py

```diff
@@ -646,6 +649,47 @@ def test_show_invites_for_guest_users(self) -> None:
         html = result.content.decode('utf-8')
         self.assertNotIn('Invite more users', html)
 
+    @override_settings(AUTHENTICATION_BACKENDS=('zproject.backends.ZulipLDAPAuthBackend',))
+    def test_show_invites_for_LDAP_only(self) -> None:
+        user_profile = self.example_user('hamlet')
+        email = user_profile.email
+
+        ldap_patcher = patch('django_auth_ldap.config.ldap.initialize')
+        self.mock_initialize = ldap_patcher.start()
+        self.mock_ldap = MockLDAP()
+        self.mock_initialize.return_value = self.mock_ldap
+        self.backend = ZulipLDAPAuthBackend()
+
+        self.mock_ldap.directory = {
+            'uid=hamlet,ou=users,dc=zulip,dc=com': {
+                'userPassword': ['testing', ]
+            }
+        }
+
+        realm = user_profile.realm
+        realm.invite_required = True
+        realm.save()
```

Comment:
Use the `do_change_` method from `zerver/lib/actions.py` for this change.  We prefer top use the standard functions rather than just editing fields in this sort of test because it's more robust to future changes in how things work.

Reply:
There doesn't appear to be a `do_change_invite_required` function in the `zerver/lib/actions.py` file. Should I create one?

Reply:
I found the function and updated the test to use it!

Reply:
Great, thanks!  Yeah I wrote that function name off memory.